packages/styles/icons/svg/IBM-8bar-logo--h23.svg

```diff
@@ -0,0 +1,8 @@
+        </g>
+    </g>
+</svg>
```

Comment:
@kennylam you will need to add the `icons` folder under `package.json` to be available in node_modules.

Reply:
```
"files": [
    "dist/**/*",
    "icons/**/*",
    "scss/**/*"
  ],
```